manifest.yml

```diff
@@ -0,0 +1,12 @@
+---
+applications:
+- name: funding-service-design-form-runner
+  docker:
+    image: ghcr.io/communitiesuk/digital-form-builder/runner:latest
+  memory: 528M
+  sandbox: true
+  routes:
+    - route: forms.dev.fundingservice.co.uk
+  env:
+    PREVIEW_MODE: TRUE
+    
```

Comment:
Can we rebase instead? This file is already on `main` branch.

Reply:
Thanks Gideon for helping with rebasing 🌟